tests/e2escenarios/e2e_images_test.go

```diff
@@ -89,35 +89,35 @@ var _ = Describe("odo supported images e2e tests", func() {
 		})
 
 		It("Should be able to verify the nodejs-10 image", func() {
-			oc.ImportImageFromRegistry("registry.access.redhat.com", filepath.Join("rhoar-nodejs", "nodejs-10:latest"), "nodejs:latest", commonVar.Project)
-			verifySupportedImage(filepath.Join("rhoar-nodejs", "nodejs-10:latest"), "nodejs", "nodejs:latest", commonVar.Project, appName, commonVar.Context)
+			oc.ImportImageFromRegistry("registry.access.redhat.com", "rhoar-nodejs"+"/"+"nodejs-10:latest", "nodejs:latest", commonVar.Project)
```

Comment:
Why are you using `+` to concatenate two strings like this?

Reply:
I miss understood the above comment, Just providing the whole image name as a string is okay but in the helper function well will require to concatenate should I use string.join?

Reply:
just do `"rhoar-nodejs/nodejs-10:latest"`